doc/sphinx/source/faq.rst

```diff
@@ -59,13 +59,6 @@ This is a useful functionality because it allows the user to `fix` things on-the
 quitting the Ipython console, code execution continues as per normal.
 
 
-Use multiple config-user.yml files
```

Comment:
Would it be worth updating this to describe how multiple configuration directories can be used? Or link the the ESMValCore docs where that is explained?

Reply:
Good point, added in https://github.com/ESMValGroup/ESMValTool/pull/3761/commits/f31f9b77fd2b97b09072d67a31feea15c274508c